### PR TITLE
Update personal portfolio page

### LINKED ISSUE #221: Update Zong Lun's Personal Portfolio Page

### PR DESCRIPTION
**Pull request by zonglun99**
- This PR fixes #221 
- Refactor some messages in parser class
- Update DG manual testing

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -525,13 +525,13 @@ This CLI based application hopes to automate a lot of the tasks users have relat
...
 
-1. Fridget should operate on most OS with Java `11` installed.
+1. Fridget should operate on common OS with Java `11` installed.
 2. A user with a fast typing speed should be able to fully optimize the usage of Fridget.
-3. Coloured dates should be able to display on most OS.
 
 ## Glossary
 
-* **OS**: Windows, Linux, OS X
+* **Common OS**: Windows, Linux, OS X
```
Comment (alvynben) [inline docs/DeveloperGuide.md:533]: Nice, thanks for this.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -549,15 +549,19 @@ This CLI based application hopes to automate a lot of the tasks users have relat
...
 2. Test cases: 
-   1. `add bacon /2021-12-12`
+   1. `add bacon /2021-12-12` followed by `1`
       Expected: Message will be printed to inform you item is added. Type command `list` to view the item added to Fridget.
-   2. `add bacon`
+   2. `add bacon /2021-12-12` followed by `5`
+      Expected: Message will be printed to inform you the increase in number of items added from 1 to 6 quantity.
+   3. `add burger /2021-12-12 ; chicken /2021-12-12` followed by `1` and `1`
+      Expected: Message will be printed to inform you that both items are added.
+   4. `add bacon`
```
Comment (alvynben) [inline docs/DeveloperGuide.md:558]: Wah fast game, awesome

File: docs/team/zonglun.md [hunk 3]
```diff
@@ -1,6 +1,49 @@
...
+all their expiring items (within 7 days) and discard any expired items in the Fridge.
+- Justification: This feature improves the product significantly because a user using this product will want to be reminded
+when their items are expiring or expired to better manage their fridge and reduce food waste.
+- Highlights: 
+
+#### Code contributed: [RepoSense](https://nus-cs2113-ay2122s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-25&tabOpen=true&tabType=authorship&tabAuthor=zonglun99&tabRepo=AY2122S1-CS2113T-W12-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+ 
+#### Project Management:
+- Created the skeleton code to kick-start the project.
+- Vetted and commented on most of the pull requests from my teammates.
```
Comment (alvynben) [inline docs/team/zonglun.md:34]: Can I say an amen!

File: src/main/java/seedu/parser/Parser.java [hunk 4]
```diff
@@ -30,6 +30,12 @@
     private static final String DATE_FORMAT = "Please try this format for the date:\n\n"
             + "    /YYYY-MM-DD\n"
             + "    Example: '... /2022-08-03";
+    private static final String INVALID_COMMAND_MESSAGE = "No command found!\n"
+            + "Enter help if you need the list of available commands.";
+    private static final String NAME_ERROR_MESSAGE = "You are not able to use '/' and '|' in item name.";
+    private static final String UPDATE_ERROR_MESSAGE = "Update quantity cannot be set to negative values.\n"
+            + "Command has been shutdown.";
+    private static final String ABORT_MESSAGE = "You have opted to abort. Command has been shutdown.";
```
Comment (alvynben) [inline src/main/java/seedu/parser/Parser.java:38]: Nice thinking.

**Review by alvynben — APPROVED** [review]

File: docs/team/zonglun.md (hunk 3 above)
Comment (zonglun99) [reply docs/team/zonglun.md:34]: amen

**Pull request merged**